Makefile

```diff
@@ -3,21 +3,56 @@ SHELL := /bin/bash # Use bash syntax
 # Set up variables
 GO111MODULE=on
 
-# Build ldflags
-VERSION ?= "v0.0.0"
-GITCOMMIT=$(shell git rev-parse HEAD)
-BUILDDATE=$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
-GO_LDFLAGS=-ldflags "-X main.version=$(VERSION) \
-			-X main.buildHash=$(GITCOMMIT) \
-			-X main.buildDate=$(BUILDDATE)"
+GO_CMD_FLAGS=-tags codegen
```

Comment:
General comment for these environment variables:
Could you please separate these out (with line breaks between them) into some general categories. Maybe:
- Input variables
- Directories and paths
- Version numbers
- Makefile-specific flags (namely `.DEFAULT_GOAL`)